api/types/cluster_manager.go

```diff
@@ -71,8 +78,7 @@ type ClusterManagerPostStatus struct {
 	CPULoad15         string               `json:"cpu_load_15"`
 	MemoryTotalAmount int64                `json:"memory_total_amount"`
 	MemoryUsage       int64                `json:"memory_usage"`
-	DiskTotalSize     int64                `json:"disk_total_size"`
-	DiskUsage         int64                `json:"disk_usage"`
+	StoragePoolUsages []StoragePoolUsage   `json:"storage_pool_usages"`
 	MemberStatuses    []StatusDistribution `json:"member_statuses"`
```

Comment:
Should storage pool usages be in `MemberStatus` as local pools will have different usage per-member?